src/applications/pensions/config/chapters/03-health-and-employment-information/helpers.js

```diff
@@ -0,0 +1,37 @@
+import moment from 'moment/moment';
+
+export function hasFederalTreatmentHistory(formData) {
+  return formData.federalTreatmentHistory === true;
+}
+export function hasNoSocialSecurityDisability(formData) {
+  return formData.socialSecurityDisability === false;
+}
+export function hasVaTreatmentHistory(formData) {
+  return formData.vaTreatmentHistory === true;
+}
+
+export function isInNursingHome(formData) {
+  return formData.nursingHome === true;
+}
+
+export function isUnder65(formData, currentDate) {
+  const today = currentDate || moment();
+  return (
+    today
+      .startOf('day')
+      .subtract(65, 'years')
+      .isBefore(formData.veteranDateOfBirth) || !formData.isOver65
+  );
+}
+
+export function isEmployedUnder65(formData) {
+  return formData.currentEmployment === true && isUnder65(formData);
+}
+
+export function isUnemployedUnder65(formData) {
+  return formData.currentEmployment === false && isUnder65(formData);
+}
+
+export function medicaidDoesNotCoverNursingHome(formData) {
+  return formData.nursingHome === true && formData.medicaidCoverage === false;
+}
```

Comment:
I like this pattern of having a helper file for each chapter. Since the form.js helper file was so large, it makes sense to move these functions to a more relevant file to improve maintainability and readability.

Reply:
Agreed - the helper file was getting unamanagably large